src/bitcoin-cli.cpp

```diff
@@ -94,67 +99,146 @@ static bool AppInitRPC(int argc, char* argv[])
         fprintf(stderr, "Error: Invalid combination of -regtest and -testnet.\n");
         return false;
     }
+    if (GetBoolArg("-rpcssl", false))
+    {
+        fprintf(stderr, "Error: SSL mode for RPC (-rpcssl) is no longer supported.\n");
+        return false;
+    }
     return true;
 }
 
+
+/** Reply structure for request_done to fill in */
+struct HTTPReply
+{
+    HTTPReply(): status(0), error(-1) {}
+
+    int status;
+    int error;
+    std::string body;
+};
+
+const char *http_errorstring(int code)
+{
+    switch(code) {
+#if LIBEVENT_VERSION_NUMBER >= 0x02010300
```

Comment:
We know the version is always at least this, so the conditional compilation is unnecessary.